core/src/main/java/io/aiven/klaw/service/ExportImportDataService.java

```diff
@@ -77,6 +78,10 @@ public class ExportImportDataService {
   private static final String ADMIN_CONFIG_PREFIX = "admin_config";
   private static final String KW_DATA_PREFIX = "kwdata";
   private static final String KW_REQUEST_DATA_PREFIX = "kwrequests_data";
+
+  private static final DateTimeFormatter DATE_TIME_FORMATTER =
```

Comment:
Can we define this in a common class and get the object from it instead of defining in every class ?

Reply:
not sure that this could be done for all...
The main reason is that for many classes there are different formats...
I've extracted the most common one